src/main/flight/pid.c

```diff
@@ -105,9 +105,10 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .pidSumLimitYaw = PIDSUM_LIMIT_YAW,
         .yaw_lpf_hz = 0,
         .dterm_lpf_hz = 100,    // filtering ON by default
+        .dterm_lpf_2_hz = 200,    // second PT1 ON by default
         .dterm_notch_hz = 260,
         .dterm_notch_cutoff = 160,
-        .dterm_filter_type = FILTER_BIQUAD,
+        .dterm_filter_type = FILTER_PT1,
```

Comment:
changing defaults is generally not recommended at the same time as implementation

Reply:
I second that, let's collect more report first that for most people PT1+PT1 are better than Biquad

Reply:
OK